Assets/_Scripts/BattleField.cs

```diff
@@ -24,51 +22,81 @@ public class BattleField : MonoBehaviour
         this.InitializeEnemies();
     }
 
-    public Player GetHeroPlayer() {
-        return playerParty.members[0];
+
+    // Field helper functions ===
+    public PlayerObject GetHeroPlayer() {
+        return GameManager.Instance.gameState.playerParty.GetFighterObjects<PlayerObject>()[0];
+    }
+
+    public PlayerObject[] GetPlayerObjects() {
+        return GameManager.Instance.gameState.playerParty.GetFighterObjects<PlayerObject>();
```

Comment:
There's some ambiguity on where our ground source of truth is for PlayerParty and EnemyParty. Should we be storing it in `GameManager`, vs grabbing them from `FighterSlots` vs storing this here in `BattleField.`

Reply:
What are your thoughts on where that source of truth should eventually come from?

Reply:
These are helper methods specific to the battlefield. It's just so that you don't need to remember how to use the slightly more complicated functions in Party. This comment should be done in PlayerRefactor PR though.